src/test/kotlin/no/nav/syfo/dialogmelding/DialogmeldingServiceSpek.kt

```diff
@@ -504,6 +540,42 @@ class DialogmeldingServiceSpek : Spek({
                         varselSvarDTO.tekst shouldBeEqualTo svarTekst
                     }
                 }
+                it("Oppretter varsel-svar når dialogmelding inneholder aktuell arbeidstaker, men parentRef, conversationRef og foresporsel mangler") {
```

Comment:
Dette er egentlig et krav fra EPJ at skal være med, men vi har aldri brydd oss om det. Hvis vi skal lage tråding på et senere tidspunkt så er dette en fordel at er med. Vi bør tipse EPJ om at de må sende dette med. Jeg har tatt kontakt med Sigbjørn for å finne ut av hvilken EPJ som denne Jira-saken er basert på.

Reply:
Lagt til en metrikk for dette nå.